frameworks/hdfs/tests/test_shakedown.py

```diff
@@ -14,67 +14,61 @@
 import sdk_tasks
 import sdk_upgrade
 import sdk_utils
-from tests.config import (
-    DEFAULT_HDFS_TIMEOUT,
-    DEFAULT_TASK_COUNT,
-    FOLDERED_SERVICE_NAME,
-    PACKAGE_NAME,
-    ZK_SERVICE_PATH
-)
+import tests.config
```

Comment:
Nope, was just reviewing https://github.com/mesosphere/dcos-commons/pull/1407/files right before getting to this section and wanted to ensure consistency, though the previous version seems to be less verbose so I'll revert.